django/db/models/options.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, meta, app_label=None):
         self.verbose_name_plural = None
         self.db_table = ''
         self.ordering = []
+        self._ordering_orig = []
```

Comment:
It will be better to use smth as @timgraham suggested - `ordering_overriden`. It will be `False` by default, and may change to `True` in case of `order_with_respect_to`.